CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contribute to `yatima-lang`
+
+The guidelines to contribute to this repository are:
+
+1. Start by creating an issue that describes what needs to be done. A defined scope is not only helpful for the contributor but also for the reviewers
+2. Assign yourself to the issue you're working on so others can easily find free issues to start working
+3. Expect (and ask for) at least *one* reviewer. We require an approval before merging anything, which also helps spreading awareness of what is being changed
```

Comment:
s/at least/exactly

Too many required reviewers creates a bystander effect. Only one official review is needed or expected, which other people should comment on as needed